package.json

```diff
@@ -58,7 +58,7 @@
     "test": "repo-tools test run --cmd npm -- run cover"
   },
   "dependencies": {
-    "@google-cloud/common": "^0.16.1",
+    "@google-cloud/common": "^0.16.2",
```

Comment:
Do we have common 0.16.2? I thought .1 is the latest one.

Reply:
Yeah, that's why I closed this :)